packages/cli/command_spec.pony

```diff
@@ -325,22 +343,66 @@ class val ArgSpec
     _name + "[" + _typ.string() + "]" +
       if not _required then "(=" + _default.string() + ")" else "" end
 
-type _Value is (Bool | String | I64 | F64)
+class _StringSeq is ReadSeq[String]
```

Comment:
I think I still need the wrapper class, but could replace the Array[String] with a Vec[String] to avoid the array copying.